kbs/src/policy_engine/opa/mod.rs

```diff
@@ -33,6 +35,8 @@ impl PolicyEngineInterface for Opa {
         resource_path: &str,
         input_claims: &str,
     ) -> Result<bool, KbsPolicyEngineError> {
+        RESOURCE_POLICY_EVALS.inc();
```

Comment:
We could move the two resource policy evals to a more [upper place](https://github.com/confidential-containers/trustee/blob/main/kbs/src/api_server.rs#L284-L286), as now policy engine is designed as a plugin interface. If a new plugin is introduced, it could help us save some code lines.